adds bap 2.1 packages

gitoleg opened the pull request.

gitoleg committed:
adds bap 2.1 packages

ivg reviewed (approved):
Not that I can sign on each of the 3k lines... but I approve this message :)

ivg commented on packages/bap-arm/bap-arm.2.1.0/opam:
```diff
@@ -0,0 +1,39 @@
+  "bap-c"
+]
+synopsis: "BAP ARM lifter and disassembler"
+description: """
```
is it fine that we have triple quotes here?

ivg committed:
adds the description for the symbolic executor

gitoleg committed:
fixes few bugs

ivg committed:
fixes synopsis for glibc runtime

ivg committed:
ok, the synopsis shall _not_ end with a dot. 

Synopsis (or description first line) should start with a capital and _not_ end with a dot

ivg committed:
removes the dot from the primus-random synopsis

ivg committed:
makes find-llvm more portable across OCaml versions

`String.equal` is not available on older versions of OCaml but here it
is perfectly fine to use polymorphic compare.
